src/highrl/envs/teacher_env.py

```diff
@@ -84,6 +85,14 @@ def __init__(
         self.robot_id = 0
         self.penality_time_step = 0
 
+        self.tensorboard_dir = "runs"
```

Comment:
To separate the tensorboard logs for the `teacher` and `robot` modules, rewrite this as
```python
self.tensorboard_dir = "runs/teacher"
```